ci/cron/daily-compat.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Do not run on PRs
+pr: none
+
+# Do not run on merge to master
+trigger: none
+
+# Do run on a schedule (daily)
+#
+# Note: machines are killed every day at 4AM UTC, so we need to either:
+# - run sufficiently before that that this doesn't get killed, or
+# - run sufficiently after that that machines are initialized.
+#
+# Targeting 6AM UTC seems to fit that.
+schedules:
+- cron: "0 6 * * *"
+  displayName: daily compat check
+  branches:
+    include:
+    - master
+  always: true
+
+jobs:
+  - job: compatibility
+    timeoutInMinutes: 240
+    strategy:
+      matrix:
+        linux:
+          pool: 'linux-pool'
+        macos:
+          pool: 'macOS-pool'
+        #windows:
+          #pool: 'windows-pool'
```

Comment:
Depending on what we end up doing here, it may make more sense to have Windows as a separate job than try to merge it with the others.